vm/file/manifest.rs

```diff
@@ -49,7 +53,8 @@ impl<N: Network> Manifest<N> {
     "program": "{id}",
     "version": "0.0.0",
     "description": "",
-    "license": "MIT"
+    "license": "MIT",
```

Comment:
@reviewers
This is the way developers using `Package` can specify editions and issue upgrades.
They will have to manually set the appropriate editions.
Would love to hear your thoughts on this form factor.

Reply:
I have never seen anything use `Package`, where is this used at the moment?

Reply:
snarkOS developer